server/spec/import_spec.rb

```diff
@@ -120,7 +120,8 @@ def import_and_wait
     it 'can be undone' do
       # Make a custom pool so we can be sure it does not get wiped
       # out during either the undo or a subsequent re-import:
-      prod = create_product(random_string(), random_string(), {:owner => @import_owner['key']})
+      prod_name = "custom_pool_prod-" + random_string()
```

Comment:
Just for future reference, `random_string("custom_pool_prod-")` would accomplish the same thing I believe.  Don't worry about changing it but I think the more concise usage would be preferable in the future.